examples/automm/tabular_dl/README.md

```diff
@@ -107,14 +108,42 @@ We can reproduce the following results by tuning `--embedding_arch` in `example_
 metrics | rmse | acc | acc | acc | acc | acc | acc | rmse | acc | rmse | rmse
-["linear" ] | 0.482 | 0.859 | 0.379 | 0.721 | 0.726 | 0.949 | RuntimeError | 8.891 | 0.963 | 0.769 | 0.761 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result.zip)
+["linear"] | 0.482 | 0.859 | 0.379 | 0.721 | 0.726 | 0.949 | RuntimeError | 8.891 | 0.963 | 0.769 | 0.761 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result.zip)
 ["linear", "relu"] | 0.477 | 0.859 | 0.370 | 0.721 | 0.726 | 0.951 | RuntimeError | 8.953 | 0.967 | 0.772 | 0.757 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result2.zip)
 ["linear", "leaky_relu"] | 0.473 | 0.858 | 0.370 | 0.722 | 0.725 | 0.947 | RuntimeError | 8.915 | 0.965 | 0.771 | 0.776 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result3.zip)
 ["linear", "relu", "linear"] | 0.468 | 0.858 | 0.374 | 0.721 | 0.723 | 0.951 | RuntimeError | 8.941 | 0.965 | 0.769\* | 0.770 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result4.zip)
 ["positional", "linear"] | 0.467 | 0.864 | 0.347 | 0.694 | 0.709 | 0.951 | RuntimeError | 9.120 | 0.967 | 0.773\* | 0.761 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result5.zip)
+["positional", "linear", "relu"] | 0.465 | 0.866 | 0.343 | 0.688 | 0.704 | 0.947 | RuntimeError | 9.131 | 0.967 | 0.774\* | 0.760 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result6.zip)
+["positional"] | 0.480 | 0.861 | 0.334 | 0.684 | 0.696 | 0.951 | RuntimeError | 9.189 | 0.967 | 0.774 | 0.765 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/tabular_example_result7.zip)
 
 \* denotes adjusting `env.per_gpu_batch_size` from `128` to `64` to support runing a larger model on our device.
 
+### 6. Hyper-parameter optimization for FT_Transformer
+Set `--mode` in [`example_tabular.py`](./example_tabular.py) to `single_hpo` to run the HPO for FTTransformer. 
+The search spaces for FT_transformer are as follws:
+- "model.numerical_transformer.ffn_dropout": tune.uniform(0.0, 0.5), 
+- "model.numerical_transformer.attention_dropout": tune.uniform(0.0, 0.5),
+- "model.numerical_transformer.residual_dropout": tune.uniform(0.0, 0.2),
+- "optimization.learning_rate": tune.uniform(0.00001, 0.001),
+
+with the tuning kwargs as follws:
+```python
+hyperparameter_tune_kwargs = {
+        "searcher": 'random',
+        "scheduler": 'FIFO',
+        "num_trials": 50,
+}
+```
+
+The results are as follws:
+Dataset (metric) | w/o HPO | w/ HPO | model_configs | results 
+----  | ----  | ----  | ----  | ----  
+ca (rmse) | 0.482 | 0.579 | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/hpo/ca_hpo_config.yaml) | [link](https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/hpo/ca_hpo_result.zip)
```

Comment:
we need to pay attention to the “ca” dataset. It is not reasonable why the performance decreased so much after the HPO.